orbit/cmd/desktop/desktop.go

```diff
@@ -433,6 +435,42 @@ func main() {
 	systray.Run(onReady, onExit)
 }
 
+func refreshBubble(sum fleet.DesktopSummary, selfServiceItem *systray.MenuItem, myDeviceItem *systray.MenuItem) {
```

Comment:
Nit: Rename to `refreshMenuList`? (It also adds/removes the self service option)